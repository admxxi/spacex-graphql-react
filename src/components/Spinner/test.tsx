import React from 'react'
import { Spinner } from './index'
import { render } from '@testing-library/react'

it('renders correctly', () => {
  const { container } = render(<Spinner />)
  expect(container.firstChild).toMatchSnapshot()
})
