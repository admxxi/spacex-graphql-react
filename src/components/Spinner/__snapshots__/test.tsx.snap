// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  class="Spinner__Style-z810iv-0 HothN"
>
  <section
    class="hero is-fullheight undefined"
  >
    <div
      class="hero-body has-text-centered"
    >
      <div
        class="container"
      >
        <img
          class="img"
          height="64"
          src="/spacex-clipart2.png"
          width="64"
        />
      </div>
    </div>
  </section>
</section>
`;
